src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/register/MinimalPullRequest.java

```diff
@@ -5,6 +5,12 @@
 
 import java.util.Objects;
 
+/**
+ * This class contains the bare minimum of information needed to store. It is deliberately small to reduce the
```

Comment:
nit: I think this can be changed to: "This class contains the bare minimum of information needed to store a pull request."